aea/aea.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         resources: Resources,
         loop: Optional[AbstractEventLoop] = None,
         timeout: float = 0.05,
-        execution_timeout: float = 1,
+        execution_timeout: float = 0,
```

Comment:
@marcofavorito I am by default disabling this now. We have too many skills which still need to be rewritten (all the ones using the ledger).

Reply:
sounds good